maat-orchestration/src/main/java/uk/gov/justice/laa/crime/orchestration/mapper/ApplicationTrackingMapper.java

```diff
@@ -57,27 +63,47 @@ protected Ioj buildIOJ(WorkflowRequest workflowRequest) {
                 .withIojResult(application.getIojResult())
                 .withIojReason(application.getIojResultNote())
                 .withIojAppealResult(ReviewResult.getFrom(iojAppealDTO.getAppealDecisionResult()))
+                .withIojAssessorName(getCapitalisedFullName(repOrderDTO.getUserCreatedEntity().getFirstName(), repOrderDTO.getUserCreatedEntity().getSurname()))
                 .withAppCreatedDate(toLocalDateTime(application.getDateCreated()));
 
     }
 
-    protected Passport buildPassport(WorkflowRequest workflowRequest) {
+    protected Passport buildPassport(WorkflowRequest workflowRequest, RepOrderDTO repOrderDTO) {
 
         ApplicationDTO application = workflowRequest.getApplicationDTO();
         PassportedDTO passportedDTO = application.getPassportedDTO();
+        PassportAssessmentDTO passportAssessmentDTO = null;
```

Comment:
We can add a prefix to differentiate the DTOs for readability, say applicationPassportAssessmentDTO and repOrderPassportAssesmentDTO

Reply:
Agree..This has been a bit confusing. This is an existing issue, so will create a story to refactor this